src/libslic3r/GCodeWriter.cpp

```diff
@@ -176,10 +176,15 @@ std::string GCodeWriter::set_acceleration(unsigned int acceleration)
         // This is new MarlinFirmware with separated print/retraction/travel acceleration.
         // Use M204 P, we don't want to override travel acc by M204 S (which is deprecated anyway).
         gcode << "M204 P" << acceleration;
-    } else {
-        // M204: Set default acceleration
+    } else if (FLAVOR_IS(gcfKlipper)) {
+        gcode << "SET_VELOCITY_LIMIT ACCEL_TO_DECEL=" << acceleration * 0.5;
```

Comment:
We already write gcode for klipper, why we do not write only one command "SET_VELOCITY_LIMIT ACCEL= ACCEL_TO_DECEL="